src/main/markdown/release-notes.md

```diff
@@ -67,6 +68,13 @@ The GWT Release Notes
 * * *
 
 <a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_12_1"></a> Release Notes for 2.12.1
+### Bug fixes:
+- Correct handling for SwitchStatements without a containing Block
```

Comment:
Should it be reworded to be more user-friendly?
```suggestion
- Correct handling for switch statements without containing braces
```
(not sure this suggestion is correct though)

Reply:
Easier to read, but I _think_ we want the "a" to be more clear - the {}s _outside_ the switch were required with this regression. That is, this would fail to compile

```java
if (true) 
  switch (1) {
    default: 
  }
```
but this would succeed:
```java
if (true) {
  switch (1) {
    default: 
  }
}
```
Perhaps something like "without being contained in braces" or "without being in a containing block"?

Reply:
"without being contained in braces" sounds good

(this is a bug fix anyway, so people who already faced the bug should know what it's about even with your original wording)